main.go

```diff
@@ -232,26 +334,39 @@ func getMilestones(baseURL string, token string, projectID string, state string)
 }
 
 // CreateMilestoneData creates new milestones with title and due date
-func createMilestoneData(advance int, timeInterval string) map[string]milestone {
+func createMilestoneData(advance int, timeInterval string, api string) map[string]milestone {
 	today := time.Now().Local()
 	milestones := map[string]milestone{}
 	switch {
 	case timeInterval == "daily":
 		for i := 0; i < advance; i++ {
 			var m milestone
-			dueDate := today.AddDate(0, 0, i).Format("2006-01-02")
-			title := dueDate
+			var dueDate, title string
+			switch {
```

Comment:
Create the title before the switch statement like:
`title = today.AddDate(0, 0, i).Format("2006-01-02")`

This is the same way for both APIs. Less duplication and also makes clear that the title is only by "accident" the same as dieDate for gitlab.